README.md

```diff
@@ -159,8 +156,26 @@ From project Root:
 1. Then in a separate window run & watch changes in the main application **app-ziti-console**
 
     ```bash
-    ng build ziti-console-node --watch
+    ng build ziti-console --watch --base-href '/zac/'
```

Comment:
advice for local dev with the SPA instead of deprecated node server

Reply:
Hmm I'm not sure we're quite ready to make this the standard. There's other things/conditionds that are required for a Dev to be able to use the ZAC integration that talks to a controller directly (without the node proxy)

For example, the Controller/Edge API needs to use a trusted TLS cert, or the user has to configure their browser to ignore the failed TLS handshake. Also the `/zac` baseHref only works for hosting ZAC via the controller

Reply:
I reverted those premature changes to the local dev instructions.